content/docs/user-guide/interactivity/robotics/concepts-and-components-overview.md

```diff
@@ -0,0 +1,111 @@
+---
+linkTitle: Concepts and Structure
+title: ROS 2 Concepts and Structure
+description: Understanding basic concepts and structure of the ROS 2 Gem in Open 3D Engine (O3DE).
+toc: true
+---
+
+## ROS 2 Concepts
+
+For a quick introduction to ROS 2 concepts, please refer to [ROS 2 Concepts documentation](https://docs.ros.org/en/humble/Concepts.html).
+
+## Structure and Communication
+
+The Gem creates a [ROS 2 node](https://docs.ros.org/en/humble/Tutorials/Understanding-ROS2-Nodes.html) which is directly a part of the ROS 2 ecosystem. As such, your simulation will not use any bridges to communicate and is subject to configuration through settings such as Environment Variables. It is truly a part of the ecosystem.
+
+Note that the simulation node is handled through `ROS2SystemComponent` - a singleton. However, you are free to create and use your own nodes if you need more than one.
+
+Typically, you will be creating publishers and subscriptions. This is done through [rclcpp API](https://docs.ros.org/en/humble/p/rclcpp/generated/classrclcpp_1_1Node.html#classrclcpp_1_1Node). Example:
+
+```
+auto ros2Node = ROS2Interface::Get()->GetNode();
+AZStd::string fullTopic = ROS2Names::GetNamespacedName(GetNamespace(), m_MyTopic);
+m_myPublisher = ros2Node->create_publisher<sensor_msgs::msg::PointCloud2>(fullTopic.data(), QoS());
+```
+
+Note that QoS class is a simple wrapper to [`rclcpp::QoS`](https://docs.ros.org/en/humble/p/rclcpp/generated/classrclcpp_1_1QoS.html).
+
+## Components overview
+
+- __Central Singleton__
+  - ROS2SystemComponent
```

Comment:
I think ROS2SystemComponent  should be in code style.
and others *Component too.